src/ert/ensemble_evaluator/evaluator.py

```diff
@@ -82,14 +65,20 @@ def __init__(self, ensemble: Ensemble, config: EvaluatorServerConfig):
         self._max_batch_size: int = 500
         self._batching_interval: float = 2.0
         self._complete_batch: asyncio.Event = asyncio.Event()
+        self._server_started: asyncio.Event = asyncio.Event()
+        self._clients_connected: set[bytes] = set()
+        self._clients_empty: asyncio.Event = asyncio.Event()
+        self._dispatchers_connected: set[bytes] = set()
```

Comment:
Same here, `self._dispatchers_connected` -> `self._connected_dispatchers`.

Reply:
Actually, dispatcher is a horrible name. Can we change it to fm_step_runners?

Reply:
I've decided to keep the names (for the legacy reason). when the dust settles we can rename it.